sys/riotboot/reset.c

```diff
@@ -30,3 +30,21 @@ void usb_board_reset_in_bootloader(void)
     *magic = RIOTBOOT_MAGIC;
     pm_reboot();
 }
+
+
+#if IS_USED(MODULE_RIOTBOOT_RESET) && IS_USED(MODULE_SHELL) && \
```

Comment:
There's already such shell command available in `sys/shell/commands/sc_sys.c`

Reply:
Yes I copied it from there, but I wanted it available even if there is no `shell_commands` module selected, that file is not compiled if that module is not there...